evm_loader/test_event.py

```diff
@@ -199,5 +274,37 @@ def test_events_of_different_instructions(self):
         self.assertLess(data[1], 0xd0)  # less 0xd0 - success
         self.assertEqual(data[2:34], bytes().fromhex('%064x' % 0xb)) #sum
 
+    def test_Partial_addReturn(self):
```

Comment:
Try to rewrite `test_addReturn` with Unittest Subtest feature to check full and partial execution. (https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests).